src/components/global/BaseTag.vue

```diff
@@ -0,0 +1,81 @@
+<script>
+import { defineComponent } from 'vue';
+
+import TagEllipse from '@/assets/icons/tag-ellipse.svg';
+
+export default defineComponent({
+    name: 'BaseTag',
+    components: {
+        TagEllipse,
+    },
+    props: {
+        labels: {
+            type: [String],
+            required: true,
+        },
+        colorScheme: {
+            validator(value) {
+                return ['red', 'gray', 'transparent'].includes(value);
+            },
+            required: true,
+        },
+    },
+});
+</script>
+
+<template>
+    <div 
+        class="base-tag"
+        :labels="labels"
+        :colorScheme="colorScheme"
```

Comment:
It's actually not that common to use custom attributes in html. If you need to do this it's better to use `data-` attributes that are specifically designed for it. If you want to use it for styles specifically (`colorScheme` in this case) it's much more common to use modifier classes ([see Modifier section](http://getbem.com/naming/#modifiers)). `labels` are probably not used and can be removed, right?

Reply:
`:labels` - that's an oversight on my part, the label as a prop is enough as it is interpolated inside a `<span>`.

`:colorScheme` - this is connected to this idea of styling depending on passed down attributes instead of modifiers I had before today's presentation about styling - I would like to address this issue cross project if it's OK with You ?😃

Reply:
That's your project, you don't need to ask me 🙈 

Plan your work in such a way that you have the results you want. You have to find out for yourself what works and what doesn't 😉